config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+reporting_patient_states:
+  description: "Monthly summary of a patient's information and health indicators. This table has one row per patient, per month, from the month of the patient's registration."
+  columns:
+    patient_id: "ID of the patient"
+    recorded_at: "Time (in UTC) at which the patient was registered"
+    status: "active, dead, migrated, etc"
+    gender: "Gender of the patient"
+    age: "Age of the patient as entered by a nurse"
+    age_updated_at: "Time (in UTC) at which the age was last updated"
```

Comment:
What convention can we use to denote a reference to another field? ie, "at which the `age` was last updated". It's not clear that "age" is referring to the previous field, I think.

Reply:
I've added single quotes around it, hope that's understood. I also wonder if we can remove this field if the `current_age` will suffice. This documentation has got me thinking about trimming some fields for clarity.

Reply:
Ha, that's a good sign. We can also choose to leave all the fields and descriptions as is, and publicize only a select few in whatever learning material we craft for epis.